chiselFrontend/src/main/scala/chisel3/core/Aggregate.scala

```diff
@@ -10,11 +10,12 @@ import chisel3.internal._
 import chisel3.internal.Builder.pushCommand
 import chisel3.internal.firrtl._
 import chisel3.internal.sourceinfo.{SourceInfo, DeprecatedSourceInfo, VecTransform, SourceInfoTransform}
+import chisel3.NotStrict.CompileOptions
```

Comment:
Initially the assumption was that all internal connections should use strict checking, but the internal connections generated by the doconnect() method called from Vec.do_apply() generated errors compiling rocket-chip with hwacha.

The initial compileOptions checks were applied at the time the containing module was constructed, but it was felt this was problematic since the same connection code would pass or fail depending on attributes specified in a different source file.